unblob/handlers/archive/tar.py

```diff
@@ -136,6 +136,36 @@ def calculate_chunk(self, file: File, start_offset: int) -> Optional[ValidChunk]
         header_size = snull(header.size)
         decode_int(header_size, 8)
 
+        def signed_sum(octets) -> int:
+            return sum(b if b < 128 else 256 - b for b in octets)
+
+        if header.chksum[6:8] not in (b"\x00 ", b" \x00"):
+            logger.error(
+                "Tar handler: invalid checksum format",
```

Comment:
Nitpick, sidetrack: we should actually bind the handler name in `_calculate_chunk`, so that all logs contain it from every handler.

Here in this PR I'd add `handler=self.NAME`, instead of this ad-hoc namespacing.